Automattic/jetpack
# Launchpad: Remove launchpad/checklist endpoint

Fixes https://github.com/Automattic/wp-calypso/issues/76368

## Proposed changes:

* Removes unused endpoint `launchpad/checklist`, which has been deprecated in favor of simply `launchpad`. All hooks/references to this endpoint are no longer used.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR to your WordPress.com sandbox and sandbox `public-api.wordpress.com`
* Create a new free site from `/start` with the Write & Publish intent.
* Skip to dashboard to load Launchpad in Stepper. Ensure it still loads without errors, and that there are no calls to `launchpad/checklist` in the Network tab
* Skip to dashboard again to go to Calypso. Open Inline Help and ensure the Launchpad status area loads without errors, and there are no calls to `launchpad/checklist` in the Network tab.
